M2/Macaulay2/packages/Macaulay2Doc/doc13.m2

```diff
@@ -225,6 +225,7 @@ document { Key => {applicationDirectory, "application directory"},
      Headline => "the path to the user's application directory",
      Usage => "applicationDirectory()",
      Outputs => { String => "the path to the user's application directory" },
+     SourceCode => applicationDirectory,
```

Comment:
You can't put this line back in, because browsers can look only in files, not in internal strings.

Reply:
But isn't the html documentation all static and generated during build?

Reply:
Yes, but is the file there at the other end of the link?  Hmm, maybe we could arrange for it to be there...

Reply:
Do you mean a link to the source file? Right now, there is no link.  It's just the output of `code` inside a `<pre>` environment.

Reply:
Oh, you're right, thanks.  It also appears that way here: https://faculty.math.illinois.edu/Macaulay2/doc/Macaulay2-1.15/share/doc/Macaulay2/Macaulay2Doc/html/_application__Directory.html

Reply:
Ack, I just noticed that `SeeAlso => applicationDirectorySuffix` needs to be changed to  `SeeAlso => "applicationDirectorySuffix"`.  My mistake from long long ago.  Would you like to add that, too?

Reply:
Sure thing!